### PR TITLE
feat: Add prebuilt @github1s/vscode-web

### PR DESCRIPTION
**Pull request by xcv58**
In this PR, we did the following:

1. Create a subdirectory `vscode-web-github1s` to deal with the official VS Code repo
2. Move the `src` to `vscode-web-github1s`.
3. Create the `@github1s/vscode-web` NPM package to publish the VS Code artifact, and use it in `package.json` so that we could develop GitHub1s without building VS Code.
4. Change the default `yarn watch` to use the NPM package, and add `yarn watch-with-vscode` for tasks that need to custom VS Code.
5. Update README and `guide.md` to reflect this change.

### CODE REVIEW

File: scripts/package/copy-extensions.sh [hunk 1]
```diff
@@ -4,19 +4,10 @@ set -euo pipefail
...
-		yarn gulp compile-web
-	fi
-}
-
 function main() {
-	ensureBuiltinExtensitions
 	cd ${APP_ROOT}
-	mkdir -p dist/static/extensions
+	TARGET="vscode-web-github1s/dist"
+	mkdir -p "${TARGET}/extensions"
```
Comment (conwnet) [inline scripts/package/copy-extensions.sh:10]: What does this line do?

**Review by conwnet — APPROVED** [review]
LGTM. Thank you so much!

File: scripts/package/copy-extensions.sh (hunk 1 above)
Comment (Siddhant-K-code) [reply scripts/package/copy-extensions.sh:10]: i think it creates a directory for the extensions at the required path.

**Review by Siddhant-K-code — APPROVED** [review]
LGTM! Amazing Work Buddy ❤️

**Review by conwnet — APPROVED** [review]

**Pull request merged**